tests/features/paragraphs/code_paragraphs.feature

```diff
@@ -0,0 +1,23 @@
+@openy @api @code-paragraph
+Feature: Code Paragraphs
+  Check Code paragraphs can be created and displayed
+
+  Background: Create basic landing page
+    Given I am logged in as a user with the "Editor" role
```

Comment:
I guess no need to login as Editor and logout in this case.
Because those further functions are using Drupal API.  And not care about roles.

Reply:
Done